content/internal/campaigns.md

```diff
@@ -31,3 +31,25 @@ Defining custom totals filters:
 Supported params available at [Fitness leaderboard](/search-fitness-activities/)
 
     {"campaign":{"fitness_total_options":{"start_at":"2016-01-01","end_at":"2019-01-01"}}}
+
+### Other campaign options
+``disable_tracker_activities``
```

Comment:
Two "``" are better than one "`" ? (... I thought one was enough, and 3 for a black comment).  (obvs not a large deal :)